cabal.project

```diff
@@ -27,23 +27,53 @@ packages:
 tests: True
 benchmarks: True
 
-package cardano-ping
+package cardano-ledger-core
+  flags: +asserts
+
+package cardano-ledger-shelley
+  flags: +asserts
```

Comment:
Do we actually want this? E.g. it could be that this impacts performance significantly, in particular for db-analyser?

Reply:
I will add a note in the `db-analyser` and `db-synthesiser documentation stating that people should disable these asserts if they build these tools in our repo. If they build it outside our repo then this will not be a problem.

Reply:
https://github.com/input-output-hk/ouroboros-consensus/pull/51